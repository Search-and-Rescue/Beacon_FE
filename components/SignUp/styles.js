import { StyleSheet } from "react-native";

const styles = StyleSheet.create({
  loginContainer: {},
  input: {},
  loginButton: {}
});

export default styles;
