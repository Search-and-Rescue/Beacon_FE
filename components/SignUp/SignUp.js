import React, { Component } from "react";
import { View, Modal, TextInput, Button } from "react-native";
import styles from "./styles";

class SignUp extends Component {
  render() {
    return (
      <Modal>
        <View style={styles.signupContainer}>
          <TextInput placeholder="Email" style={styles.input} />
          <TextInput placeholder="Password" style={styles.input} />
          <View style={styles.signupButton}>
            <Button title="signup" />
          </View>
        </View>
      </Modal>
    );
  }
}

export default SignUp;
