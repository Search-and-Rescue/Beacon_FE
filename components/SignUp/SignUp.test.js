import "react-native";
import React from "react";
import { shallow } from "enzyme";
import SignUp from "./SignUp";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("SignUp", () => {
  it("should match the snapshot with all of the data passed through", () => {
    expect(shallow(<SignUp />)).toMatchSnapshot();
  });
});
