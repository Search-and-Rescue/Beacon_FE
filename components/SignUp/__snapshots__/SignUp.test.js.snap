// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp should match the snapshot with all of the data passed through 1`] = `
<Component
  hardwareAccelerated={false}
  visible={true}
>
  <View>
    <TextInput
      allowFontScaling={true}
      placeholder="Email"
      rejectResponderTermination={true}
      style={Object {}}
      underlineColorAndroid="transparent"
    />
    <TextInput
      allowFontScaling={true}
      placeholder="Password"
      rejectResponderTermination={true}
      style={Object {}}
      underlineColorAndroid="transparent"
    />
    <View>
      <Button
        title="signup"
      />
    </View>
  </View>
</Component>
`;
